src/tiled/mainwindow.ui

```diff
@@ -156,6 +156,7 @@
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="actionZoomNormal"/>
+    <addaction name="actionZoom_With_Ctrl"/>
```

Comment:
It's rare that a user will change this setting, so please don't add it to the menu. Add the checkbox to the Preferences dialog instead.